docs/Readme.md

```diff
@@ -27,24 +27,36 @@
     [Heuristic](Learning-Environment-Design-Heuristic-Brains.md),
     [Learning](Learning-Environment-Design-Learning-Brains.md)
 * [Learning Environment Best Practices](Learning-Environment-Best-Practices.md)
-* [Using the Monitor](Feature-Monitor.md)
-* [Using the Video Recorder](https://github.com/Unity-Technologies/video-recorder)
-* [Using an Executable Environment](Learning-Environment-Executable.md)
-* [Creating Custom Protobuf Messages](Creating-Custom-Protobuf-Messages.md)
+
+### Advanced Usage
+  * [Using the Monitor](Feature-Monitor.md)
+  * [Using the Video Recorder](https://github.com/Unity-Technologies/video-recorder)
+  * [Using an Executable Environment](Learning-Environment-Executable.md)
+  * [Creating Custom Protobuf Messages](Creating-Custom-Protobuf-Messages.md)
 
 ## Training
 
 * [Training ML-Agents](Training-ML-Agents.md)
+* [Using TensorBoard to Observe Training](Using-Tensorboard.md)
 * [Training with Proximal Policy Optimization](Training-PPO.md)
 * [Training with Soft Actor-Critic](Training-SAC.md)
+
+### Advanced Training Methods
+
 * [Training with Curriculum Learning](Training-Curriculum-Learning.md)
 * [Training with Imitation Learning](Training-Imitation-Learning.md)
 * [Training with LSTM](Feature-Memory.md)
 * [Training Generalized Reinforcement Learning Agents](Training-Generalized-Reinforcement-Learning-Agents.md)
+* [Training Using Concurrent Unity Instances](Training-Using-Concurrent-Unity-Instances.md)
+
+### Cloud Training (Deprecated)
+Here are the cloud training set-up guides for Azure and AWS. We no longer use them ourselves and 
+so they may not be work correctly. We've decided to keep them up just in case they are helpful to
+you. If you do try them out, please share your experience with us, and feel free to submit a PR
+with any necessary changes.
```

Comment:
I think our plan is to deprecate these - so we shouldn't ask for PR's to fix them. We should, in fact, say that these pages will eventually be removed. @unityjeffrey - thoughts?

Reply:
Additionally, we should add this same message to each of the AWS/Azure pages separately.

Reply:
removed line about PR's and added a note on AWS/Azure pages.